src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -490,6 +496,7 @@ mod tests {
                 data: b"".to_vec(),
                 host_nonce_commitment: None,
                 chain_id: 0,
+                case: 2,
```

Comment:
please use explicit names for clarity like you did above, i.e. ` pb::AddressCase::Mixed as _`. I consider it a big shortcoming of prost (the Rust protobuf lib we use) that enum fields are not decoded into the actual type but into i32 :( See

https://github.com/tokio-rs/prost/issues/276
https://github.com/tokio-rs/prost/issues/303